hello
Traceback (most recent call last):
  File "Testing.py", line 8, in <module>
    with open('HELLO.txt') as file:
FileNotFoundError: [Errno 2] No such file or directory: 'HELLO.txt'
